inference-engine/tests_deprecated/helpers/CMakeLists.txt

```diff
@@ -37,11 +37,11 @@ endfunction()
 
 add_helpers(${TARGET_NAME})
 
-target_link_libraries(${TARGET_NAME} PUBLIC commonTestUtils)
+target_link_libraries(${TARGET_NAME} PUBLIC commonTestUtils gmock gtest)
```

Comment:
`gmock` and `gtest` is necessary here?

Reply:
@iefode `gmock` only

Reply:
`gmock` is linked to the `unit_test_utils`. I think it is extra for `ie_helpers`